test/jsonLong.spec.js

```diff
@@ -19,4 +19,17 @@ describe("Tests the Long support", function(){
         expect(expectedObject).to.equal(input);
         done();
     });
+
+    it("tests the JSON parse and stringify functionality with unsigned support", function(done){
+        var parsedObject = JSONLong.parse(input, {unsigned: true});
+        expect(parsedObject.number.toString()).to.equal("123");
```

Comment:
what's the difference between this test and the earlier one?

what exactly is the change in behavior that having {unsigned: true} cause?

Reply:
I think I was supposed to use a number bigger than `9223372036854775807` to make sure that it can convert unsigned long values. I will improve the test.